### DISCUSSION TITLE
What is the root frequency distribution and how to determine which to use

### DISCUSSION THREAD

**Discussion by tsa4a12** [hahnlab/CAFE5#191] [Q&A] [answered]
In the params section it stated that
'''
--poisson, -p

Use a Poisson distribution for the root frequency distribution. If no -p flag is given, a uniform distribution will be used. A value can be specified (-p10, or --poisson=10); otherwise the distribution will be estimated from the gene families.
'''

How do I know when to use it or when not to use it?
I suppose some assumptions were made on the tree but I am not an expert in terms of details in construction of a phylogenetic tree...

Thanks,
Tom

**Comment by benfulton** [comment, marked as answer]
The root frequency distribution is used to determine the prior probability for the likelihood calculation. Dr. Hahn recommends not using this option any longer, however; the default value should be sufficient.
